### PR TITLE
Update notebooks

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

**Comment by ocaisa** [comment]
@AdamWlodarczyk @milBia Let's update these notebooks first and I will make a new PR for the rest

File: 04-environment.ipynb [hunk 1]
```diff
@@ -2,236 +2,204 @@
...
    "source": [
-    "# Complex computing environments\n",
-    "Now let's imagine some non-trivial tasks that require specific environments in which to compute "
+    "# A \"real life\" example"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "limiting-pension",
```
Comment (AdamWlodarczyk) [inline 04-environment.ipynb:14]: Is this some autogenerated thing from jupyter?

File: 04-environment.ipynb [hunk 2]
```diff
@@ -2,236 +2,204 @@
...
+    ")\n",
+    "from jobqueue_features import on_cluster, mpi_task, get_task_mpi_comm\n",
+    "from jobqueue_features.clusters_controller import (\n",
+    "    clusters_controller_singleton as controller,\n",
+    ")"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "endangered-saint",
```
Comment (AdamWlodarczyk) [inline 04-environment.ipynb:31]: :open_mouth:

**Comment by ocaisa** [comment]
I'm going to merge this because it is hard for me to work on updates while other PRs are open

File: 04-environment.ipynb (hunk 1 above)
Comment (ocaisa) [reply 04-environment.ipynb:14]: Must be, I am certainly not going to the bother of doing that!

**Pull request merged**